internal/db/schema/postgres/postgres.go

```diff
@@ -228,16 +278,24 @@ func runesLastIndex(input []rune, target rune) int {
 
 // SetVersion sets the version number, and whether the database is in a dirty state.
 // A version value of -1 indicates no version is set.
-func (p *Postgres) SetVersion(ctx context.Context, version int, dirty bool) error {
+func (p *Postgres) setVersion(ctx context.Context, version int, dirty bool) error {
```

Comment:
It would be helpful to update the first word of the function doc from `SetVersion` to `setVersion` to reflect the name.